deploy/test/Dockerfile

```diff
@@ -24,3 +24,10 @@ RUN mkdir -p ocbin && \
     tar xvf oc.tar.gz --strip-components=1 -C ocbin && \
     mv ocbin/oc /usr/local/bin/oc && \
     rm -rf ocbin oc.tar.gz
+
+# Install Helm
+RUN curl https://baltocdn.com/helm/signing.asc | sudo apt-key add - && \
+    sudo apt-get install apt-transport-https --yes && \
+    echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list &&\
+    sudo apt-get update && \
+    sudo apt-get install helm=3.2.*
```

Comment:
Why do you need `sudo`? previous lines did not require `sudo`.

Reply:
see here: https://helm.sh/docs/intro/install/#from-apt-debianubuntu took it from their official docs page

Reply:
why do we need to install apt-transport-https ?

Reply:
removed sudo
@oburstein-hub this was taken from helm page  https://helm.sh/docs/intro/install/#from-apt-debianubuntu